soccer/planning/tests/PlannerTest.cpp

```diff
@@ -298,8 +300,8 @@ TEST(Planning, settle_random) {
     int failure_count = 0;
 
     for (int i = 0; i < 500; i++) {
-        world_state.ball.position = Point{random(-1.5, 1.5), random(2, 4)};
-        world_state.ball.velocity = Point{random(-.3, .3), random(-0.3, -.2)};
+        world_state.ball.position = Point{random(-1.5, 1.5), random(3, 4)};
```

Comment:
I know this was mentioned in the future, but is it planned to make this deterministic?

From what I can tell, as long as the big compilers are used and the specific prng generator is specified, it should be deterministic across machines. 

Specifically the example here with mt19937 should work if we set a static see
https://en.cppreference.com/w/cpp/numeric/random/uniform_real_distribution

Reply:
Yeah, setting a deterministic seed is the easy fix.